### ISSUE TITLE
Use named arguments instead of a predefined sequence for RPC authentication

### ISSUE THREAD

**Issue by arik-so** [lightningdevkit/ldk-sample#50]
Perhaps use a library such as https://github.com/tailhook/rust-argparse? That way, arguments can be provided in any order rather than requiring adherence to a prescribed sequence.

**Comment by TheBlueMatt** [comment]
We could change the parsing, yea, but lets please not take a dependency for something like that, never.

**Comment by arik-so** [comment]
Fair enough, no dependencies. Also, keep in mind these are all extremely low priority suggestions. I'm just putting this here to keep track of minor improvement potential.
